src/PackageDetailsViewExtension/PackageDetailsViewExtension.csproj

```diff
@@ -48,6 +48,7 @@
       <Name>DynamoServices</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\PythonMigrationViewExtension\PythonMigrationViewExtension.csproj" />
```

Comment:
What is the purpose of this new dependency?

Reply:
Not sure why this change was added. I didn't change this at least manually. I'm gonna removed that